Flow.Launcher/SettingWindow.xaml

```diff
@@ -323,8 +323,8 @@
             <Setter Property="Padding" Value="0,0,0,0" />
             <Setter Property="Margin" Value="0,0,8,5" />
             <Setter Property="BorderBrush" Value="{DynamicResource Color03B}" />
-            <Setter Property="MinWidth" Value="330" />
-            <Setter Property="Height" Value="98" />
+            <Setter Property="MinWidth" Value="100" />
+            <Setter Property="Height" Value="Auto" />
```

Comment:
Do we need to change this? ATM when you open settings for the first time on a brand new install, the Settings window is opening to the top left corner instead of center.

Reply:
Useless Value. Removed.